specification/quantum/resource-manager/Microsoft.Quantum/preview/2019-11-04-preview/quantum.json

```diff
@@ -658,6 +658,10 @@
           "type": "string",
           "description": "Description about this sku."
         },
+        "restrictedAccessUri": {
```

Comment:
Question, is this intended to be singular URI or potentially could be a collection of them?

Reply:
It’s meant to be singular, each sku can have 1 uri associated with it